### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by iurimatias**

### CODE REVIEW

File: source/docs/plugin_reference.md [hunk 1]
```diff
@@ -20,71 +19,61 @@ This call will return the content of the current asset file so the plugin can tr
 ``matchingFiles`` is an array of matching files the plugin should be called for e.g [``**/*.js``, ``!vendor/jquery.js``] matches all javascript files except vendor/jquery.js
 
 options available:
- * targetFile - filename to be generated
- * source - content of the file
+ * `targetFile - filename to be generated
+ * `source - content of the file
```
Comment (jrainville) [inline source/docs/plugin_reference.md]: You didn't close the back-tick

File: themes/navy/layout/partial/after_footer.swig [hunk 2]
```diff
@@ -15,6 +16,28 @@
...
+function highlight() {
+  $('pre code').each(function(i, block) {
+    hljs.highlightBlock(block);
+  });
+}
+
+$(document).ready(function() {
+    try {
+      highlight();
+    } catch(err) {
```
Comment (jrainville) [inline themes/navy/layout/partial/after_footer.swig]: Why would it fail?

File: themes/navy/source/css/_partial/main.styl [hunk 3]
```diff
@@ -1,10 +1,10 @@
 @import url("https://fonts.googleapis.com/css?family=Roboto+Mono");
-html, body, div, span, applet, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, a, abbr, acronym, address, big, cite, code, del, dfn, em, img, ins, kbd, q, s, samp, small, strike, strong, sub, sup, tt, var, b, u, i, center, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, canvas, details, embed, figure, figcaption, footer, header, hgroup, menu, nav, output, ruby, section, summary, time, mark, audio, video {
+html, body, div, span, applet, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, a, abbr, acronym, address, big, cite, del, dfn, em, img, ins, kbd, q, s, samp, small, strike, strong, sub, sup, tt, var, b, u, i, center, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, canvas, details, embed, figure, figcaption, footer, header, hgroup, menu, nav, output, ruby, section, summary, time, mark, audio, video {
   margin: 0;
   padding: 0;
   border: 0;
   font-size: 100%;
-  font: inherit;
+  //font: inherit;
```
Comment (jrainville) [inline themes/navy/source/css/_partial/main.styl]: Can remove? What did it do?

File: themes/navy/layout/partial/after_footer.swig (hunk 2 above)
Comment (iurimatias) [reply themes/navy/layout/partial/after_footer.swig]: it fails sometimes, seems to be a known issue, document ready is suppose to prevent it but still

**Pull request merged**